examples/basic_authentication/lib/user_repository.dart

```diff
@@ -0,0 +1,120 @@
+import 'dart:convert';
+
+import 'package:crypto/crypto.dart';
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+@visibleForTesting
+
+/// In memory database of users.
```

Comment:
Nit: avoid dangling tags
```suggestion
/// In memory database of users.
@visibleForTesting
```